docs/wallets/ledger_desktop_wallet.md

```diff
@@ -50,33 +52,56 @@ After the app is set, please conducting the following steps to create a wallet w
+        <img src="./assets/ledger_desktop_wallet/s2-5-4.png" />    
+        Click "Connect". Both your Crypto.org native chain & Cronos Chain wallet should be successfully created.
+        <img src="./assets/ledger_desktop_wallet/s2-5-5.png" />    
+
 
-- **Step 2.5** - Before clicking "*Create Wallet*", make sure you have unlocked and started your Crypto.org Chain App on your Ledger device. Also, you should be able to see "*Crypto.org Chain ready*" displayed on your device as follow: 
-        <img src="./assets/ledger_desktop_wallet/s2-5.jpg" />    
-    At this step, your wallet should be successfully created:
-            <img src="./assets/ledger_desktop_wallet/s2-5-1.png" />
+### Step 3 - Before you perform transactions with Ledger device
+Before start with the sending transaction/performing staking operations, we would need some funds to begin with:
 
+- **Step 3.1** - Go to "*Assets*" and pick the asset you would like to transfer. Go to "*Receive*" tab in the desktop wallet and your wallet address will be displayed. Depending on which asset you choose, you might need to open different App on your ledger. 
 
-### Step 3 - Create wallet in Desktop Wallet with Ledger device
-In order to start with the sending transaction/performing staking operations, we would need some funds to begin with:
+| App | Supported Assets |
+| :---: | --- | --- |
+| Crypto.org  | CRO (Crypto.org Chain) |
+| Ethereum  | CRO (Cronos Chain) |
 
-- **Step 3.1** - Go to the "*Receive*" option in the desktop wallet and your wallet address will be displayed. Verify the address by clicking "*Verify address on Ledger*", you should see the same address being displayed on your ledger device as well.
+Verify the address by clicking "*Verify address on Ledger*", you should see the same address being displayed on your ledger device as well.
+            <img src="./assets/ledger_desktop_wallet/s3-1.png" />
 
 - **Step 3.2** - Once the address has been confirmed, copy this address and request some funds from the [testnet faucet](https://crypto.org/faucet)
 
-- **Step 3.3** - Your current balance will be available on the Home page. If you would like to view the transaction details, simply hit the hash link under Transaction Hash and it will direct you to the [explorer](https://crypto.org/explorer/) where different details related to transactions will be displayed:
+| App | Supported Assets |
+| :---: | --- | --- |
+| CRO (Crypto.org Chain) | [https://crypto.org/faucet](https://crypto.org/faucet) |
+| CRO (Cronos Chain) | [https://cronos.crypto.org/faucet](https://cronos.crypto.org/faucet) |
+
+- **Step 3.3** - Your current balance will be available in "*Assets*" page. If you would like to view the transaction details, go to "*Transaction*" Tab and simply hit the hash link under Transaction Hash and it will direct you to the corresponding explorer where different details related to transactions will be displayed:
```

Comment:
```suggestion
- **Step 3.3** - Your current balance will be available on the "Assets" page. If you would like to view the transaction details, go to the "Transaction" Tab and simply hit the hash link under Transaction Hash, and it will direct you to the corresponding explorer where different details related to transactions will be displayed: